src/libs/TransactionUtils.js

```diff
@@ -79,7 +79,7 @@ function buildOptimisticTransaction(
  * @returns {Boolean}
  */
 function hasReceipt(transaction) {
-    return !_.isEmpty(lodashGet(transaction, 'receipt'));
+    return Boolean(lodashGet(transaction, 'receipt.receiptID'));
```

Comment:
We don't create `receiptID` [optimistically](https://github.com/Expensify/App/blob/8d5ae3a22a952677f8c48ce8a9c288f51a5eb8d0/src/libs/actions/IOU.js#L378-L384), so this change would prevent images from loading before we get a receiptID back from the client

Reply:
Can you respond to me here -> https://expensify.slack.com/archives/C057F53HR40/p1692826510517379?thread_ts=1692826029.393829&cid=C057F53HR40